core/tests/behavior/append.rs

```diff
@@ -84,15 +85,16 @@ pub async fn test_append_with_dir_path(op: Operator) -> Result<()> {
 
 /// Test append with cache control must success.
 pub async fn test_append_with_cache_control(op: Operator) -> Result<()> {
-    if !op.info().full_capability().append_with_cache_control {
+    if !op.info().full_capability().write_with_cache_control {
```

Comment:
I think we should also check the `append` capability

Reply:
Already checked.